azurelinuxagent/ga/exthandlers.py

```diff
@@ -978,33 +978,40 @@ def write_ext_handlers_status_to_info_file(self, vm_status):
             "python_version": "Python: {0}.{1}.{2}".format(PY_VERSION_MAJOR, PY_VERSION_MINOR, PY_VERSION_MICRO),
             "crp_supported_features": [name for name, _ in get_agent_supported_features_list_for_crp().items()],
             "extension_supported_features": [name for name, _ in
-                                             get_agent_supported_features_list_for_extensions().items()]
+                                             get_agent_supported_features_list_for_extensions().items()],
+            "last_successful_status_upload_time": time.strftime("%Y-%m-%dT%H:%M:%SZ", time.gmtime())
```

Comment:
Do we not log this property in the status blob? It would be preferable to get this data directly from the blob as compared to getting it from the `time` module.

From the [publshers doc](https://microsoft.sharepoint.com/:w:/t/ComputeVM/EforWBxWd9dGrdcuW732cBkBYS5xlb0UV1LzjtXb2CISUw?e=azeir0) sec 2.4.2 (Status reporting), I do see we have a property where we log the time - 

`"timestampUTC": "<current utc time>",`

We should use this property